src/Homework2/task1.py

```diff
@@ -0,0 +1,12 @@
+""" Напишите программу, которая считает общую цену.
+Вводится M рублей и N копеек цена,
+а также количество S товара Посчитайте общую цену
+в рублях и копейках за L товаров"""
+
+M = 3
+N = 20
+S = 3
+amount_RUB = M * S
+amount_CENTS = N * S
+
+print('Общая цена', str(amount_RUB), 'рублей', str(amount_CENTS), 'копеек')
```

Comment:
L=10
N=90
it would show 900 kopecks. it's strange and doesn't solve task in the description